packages/nameguard-sdk/src/index.ts

```diff
@@ -448,6 +475,8 @@ const DEFAULT_NETWORK: Network = "mainnet";
 const DEFAULT_INSPECT_LABELHASH_PARENT = ETH_TLD;
 export const DEFAULT_COMPUTE_NAMEGUARD_REPORT = false;
 const MAX_BULK_INSPECTION_NAMES = 250;
+export const MAX_INSPECTED_NAME_CHARACTERS = 200;  // includes label separators
```

Comment:
It's best to write comments like these in /* ... */ style above the related code. This is nice because then IDEs such as VSCode will show that comment anywhere in the code where you hover over the related definition.